### ISSUE TITLE
"Licenses" page has unbound translation variable

### ISSUE THREAD

**Issue by eignnx** [rust-lang/www.rust-lang.org#977]
# Design Bug Report
**Thanks to [Luna on Twitter](https://twitter.com/smolcomputergf/status/1213905572434731009) for finding this bug!**

## Page(s) Affected
Licenses page

## What needs to be fixed?
The text should say "Apache License, Version 2.0" (as per [the translation file](https://github.com/rust-lang/www.rust-lang.org/blob/fd672b4594de5ef4049dccdec1aaa72dd74bd9f4/locales/en-US/licenses.ftl#L9)), but instead says "Unknown localization licenses-apache-v-".

Screenshot:
![image](https://user-images.githubusercontent.com/34199632/71794138-26fcd280-300e-11ea-8d69-71c667f9a56a.png)

## Suggested Improvement
I think the final quote mark (`"`) in [this line](https://github.com/rust-lang/www.rust-lang.org/blob/fd672b4594de5ef4049dccdec1aaa72dd74bd9f4/templates/policies/licenses.hbs#L17) needs to be changed from this

```html
<li><a href="http://www.apache.org/licenses/LICENSE-2.0">{{fluent "licenses-apache-v-"2-0-link}}</a></li>
```

to this:

```html
<li><a href="http://www.apache.org/licenses/LICENSE-2.0">{{fluent "licenses-apache-v-2-0-link"}}</a></li>
```

Basically one of the fluent variable names was cut in half during a refactoring.
